crates/pop-cli/src/commands/up/contract.rs

```diff
@@ -381,33 +357,6 @@ impl UpContractCommand {
 			Ok((call_data, hash))
 		}
 	}
-
-	async fn wait_for_signature(&self, call_data: Vec<u8>) -> anyhow::Result<Option<String>> {
```

Comment:
Refactor: Will be used in more than one place. Moved  into a new file `common/wallet`